appscale/tools/templates/AppScalefile

```diff
@@ -0,0 +1,162 @@
+---
+
+
+##################################################
+########## Virtual Machine (VM) Layout ###########
+##################################################
+# The deployment strategy used in this AppScale deployment.
+# Each service/role can run on one or multiple machines.
+# Primary roles: master, appengine, database, zookeeper
+# Optional roles: loadbalancer, taskqueue, search
+
+# For each role other than master you can specify multiple machines.
+# Use the IPs of existing machines,
+# or node-X notation if you want AppScale to start new machines on the
+# Infrastructure of choice.
+
+ips_layout:
+  master : [vm1_private_IP | node-1]
+  appengine:
+   - [vm2_private_IP | node-2]
+   - [vm3_private_IP | node-3]
+  database:   # Cassandra needs a quorum to answer queries (hence 3)
+   - [vm4_private_IP | node-4]
+   - [vm5_private_IP | node-5]
+   - [vm6_private_IP | node-6]
+  zookeeper:  # Zookeeper needs a majority of peers online (hence 3)
+   - [vm7_private_IP | node-7]
+   - [vm8_private_IP | node-8]
+   - [vm9_private_IP | node-9]
+login : [vm1_public_IP | Domain Name]
```

Comment:
I think we should have some comment for login. Something like: Login is used to instruct AppScale which one is the visible public IP used by clients to connect. It could be the public IP of the load-balancer instance, or it could be the list of IPs used for exernal load-balancers.